core/src/main/java/io/questdb/griffin/SqlCompilerImpl.java

```diff
@@ -3295,7 +3310,7 @@ private void backupTable(@NotNull TableToken tableToken) {
                         }
                         RecordCursor cursor = reader.getCursor();
                         //statement/query timeout value  is most likely too small for backup operation
-                        copyTableData(cursor, reader.getMetadata(), backupWriter, writerMetadata, recordToRowCopier, SqlExecutionCircuitBreaker.NOOP_CIRCUIT_BREAKER);
+                        copyTableData(cursor, reader.getMetadata(), backupWriter, writerMetadata, recordToRowCopier, -1, 0, SqlExecutionCircuitBreaker.NOOP_CIRCUIT_BREAKER);
```

Comment:
we should be using defaults for backup too, even though this backup method is legacy